### PR TITLE
Fix link to DOCUMENTATION.md

### PR DESCRIPTION
**Pull request by sampart**
Now correctly points to the legacy repository rather than 404ing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ If you're starting a new integration, you should use the new [GoCardless API](ht
 
 If you're not sure whether this is the right library for you, get in touch - you can reach our developer support team at <api@gocardless.com>.
 
-If you're sure this is the library you want to be using, see [DOCUMENTATION.md](https://github.com/gocardless/gocardless-php/blob/master/DOCUMENTATION.md) for documentation and helpful resources.
+If you're sure this is the library you want to be using, see [DOCUMENTATION.md](DOCUMENTATION.md) for documentation and helpful resources.
```
Comment (timrogers) [inline README.md]: I think you'll need to add the full URL (albeit with the right repository name), else this link won't work on GitHub

**Review by timrogers — CHANGES_REQUESTED** [review]
Thanks so much :+1: Just a little tweak and then we can get this in

File: README.md (hunk 1 above)
Comment (sampart) [reply README.md]: Ah, okay.  I've amended the commit and force-pushed.

**Comment by sampart** [comment]
Looks like someone else fixed this already: https://github.com/gocardless/gocardless-legacy-php/commit/9e4e9a3c48f2885ade6463fceb8f5fc0421d3cf4

**Pull request closed without merging**